docs/6.-Engineering/Ways-Of-Working/README.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Ways of Working
+authors: 
+- Rhodh
+reviewed: 
+reviewer:
+next-review: 
+---
+# Ways of Working
+
+While we encourage empowered self organizing teams, there are some default ways of working teams are expected to adopt. These can be broken down into 5 Pillars
+
+ - Collaboration
+ - Test First Delivery
+ - Driving Delivery 
+ - Champion Quality (TBD name)
```

Comment:
How about **Ownership of Delivery** and **Quality Emphasis** to make these both nouns like the others?

On reflection I dislike "pillars" and would suggest **core principles**, but this is very much personal preferance!

Reply:
Making terms consistent is a good idea. 

I think I prefer pillars to core principles just to try and emphasise the idea that these are building blocks to empowered autonomous teams but both work tbh. 

@KarenFarrell36 @dingjimmy @beeozguler which do prefer?